frontend/src/types/crawler.ts

```diff
@@ -52,6 +52,8 @@ export type WorkflowParams = {
   crawlerChannel: string;
 };
 
+export const DEFAULT_MAX_SCALE = 3;
```

Comment:
We probably want to keep constants outside of the type files, can we move this to `@/utils/crawler`?